spec_3/StringIndexer.js

```diff
@@ -0,0 +1,18 @@
+export default class StringIndexer {
+
+  static trimSymbols( word ) {
+    return word.replace( /[.;:\-,=/\\?<>"'+!@#$%^&*(()]/, "" )
+  }
+
+  static getWord( phrase, index ) {
+    if ( index < 1 ) {
```

Comment:
Consider moving both validity checks to the top of the function - they both return the same thing, and it reads a little more concisely:
```js
let words = phrase.split( ' ' )

if( index < 1 || index > words.length ) {
  return ''
} else {
  return StringIndexer.trimSymbols( words[ index - 1 ] )
}
```